lib/game.rb

```diff
@@ -19,6 +28,19 @@ def welcome_player
   end
 
   def display_board
-    console.output(display.present)
+    board_values = board_manager.board_values
+    console.output(display.present(board_values))
+  end
+
+  def prompt_player_to_choose_space
+    console.output(prompt.choose_a_space)
+  end
+
+  def gets_move
+    @latest_move = console.gets_input.to_i
```

Comment:
Should game be responsible for changing the input to an integer?  What happens if the input is a question mark of any other non number input?

Reply:
Still working on implementing the validation! @damonkelley I pushed up a slightly spiked out version of the Prompt#call method we were talking about earlier. Is this heading in the direction you were going with it?

Reply:
Yeah, I think that is a suitable direction to take. 👍